composer.json

```
@@ -18,8 +18,8 @@
     ],
     "require": {
         "php": ">=7.2",
-        "illuminate/support": "~6.0",
-        "illuminate/database": "~6.0"
+        "illuminate/support": "~7.0",
+        "illuminate/database": "~7.0"
```

This should be:

```suggestion
        "illuminate/support": "^6.0|^7.0",
        "illuminate/database": "^6.0|^7.0""
```

So it stays available for any 6.* or 7.* version.

---

You are right, my bad.